docroot/profiles/favrskov/themes/favrskovtheme/template/Views/job/views-view-table--job--job-listing-by-table.tpl.php

```diff
@@ -41,23 +44,21 @@
       <?php $destination_link = $row['field_job_destination_link']; ?>
       <?php unset($row['field_job_destination_link']); ?>
     <?php endif; ?>
-
-    <tr <?php if ($row_classes[$row_count]) { print 'class="' . implode(' ', $row_classes[$row_count]) .'"';  } ?>>
+    <tr style="cursor: pointer;" class='clickable-row' data-href='<?php print $destination_link; ?>' <?php if ($row_classes[$row_count]) { print 'class="' . implode(' ', $row_classes[$row_count]) .'"';  } ?>>
       <?php foreach ($row as $field => $content): ?>
         <td <?php if ($field_classes[$field][$row_count]) { print 'class="'. $field_classes[$field][$row_count] . '" '; } ?><?php print drupal_attributes($field_attributes[$field][$row_count]); ?>>
-          <?php if (!empty($destination_link) && $field_classes[$field][$row_count] == 'views-field views-field-title') : ?>
-            <a href="<?php print $destination_link; ?>">
-          <?php endif; ?>
-
           <?php print $content; ?>
-
-          <?php if (!empty($destination_link)) : ?>
-            </a>
-          <?php endif; ?>
         </td>
-
       <?php endforeach; ?>
     </tr>
   <?php endforeach; ?>
   </tbody>
 </table>
+
+<?php
```

Comment:
Please move out the JS in separate file. You can create js/ folder in custom module folder and then attach the specific file only when it is required and we access the job listing page. This is the right way to do it. JS should not reside in the template.

Reply:
JS moved in external file in favrskov theme and added only to current view.